collector/mssql.go

```diff
@@ -1430,6 +1452,12 @@ func NewMSSQLCollector() (Collector, error) {
 			[]string{"instance", "resource"},
 			nil,
 		),
+		LocksAverageWaitTimems_Base: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "locks_average_wait_seconds_lookups"),
```

Comment:
This isn't clear to me - what does this mean?

Reply:
As there is no description in the MS [documentation](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-locks-object?view=sqlallproducts-allversions) I can only guess that this is are the number of lock events.

Reply:
Ok, so then we'd have `lock_wait_seconds_total` and `lock_waited_total` or something similar in naming?